testing/unit_tests/test_slice_filter.py

```diff
@@ -88,5 +89,58 @@ def test_slice_filter(download_data_testing_test_files, transforms_dict, train_l
         assert cmpt_neg != 0 and cmpt_pos != 0
 
 
+@pytest.mark.parametrize('transforms_dict', [{"CenterCrop": {"size": [128, 128], "applied_to": ["im", "gt"]}}])
+@pytest.mark.parametrize('train_lst', [['sub-rat3_ses-01_sample-data9_SEM.png']])
+@pytest.mark.parametrize('target_lst', [["_seg-axon-manual", "_seg-myelin-manual"]])
+@pytest.mark.parametrize('slice_filter_params', [
+    {"filter_empty_mask_patch": False, "filter_empty_input_patch":  True},
+    {"filter_empty_mask_patch": True, "filter_empty_input_patch": True}])
+@pytest.mark.parametrize('dataset_type', ["training", "testing"])
+def test_patch_filter(download_data_testing_test_files, transforms_dict, train_lst, target_lst, slice_filter_params,
+    dataset_type):
+
+    cuda_available, device = imed_utils.define_device(GPU_ID)
+
+    loader_params = {
+        "transforms_params": transforms_dict,
+        "data_list": train_lst,
+        "dataset_type": dataset_type,
+        "requires_undo": False,
+        "contrast_params": {"contrast_lst": ['SEM'], "balance": {}},
+        "path_data": [os.path.join(__data_testing_dir__, "microscopy_png")],
+        "bids_config": f"{path_repo_root}/ivadomed/config/config_bids.json",
+        "target_suffix": target_lst,
+        "extensions": [".png"],
+        "roi_params": {"suffix": None, "slice_filter_roi": None},
+        "model_params": {"name": "Unet", "length_2D": [32, 32], "stride_2D": [32, 32]},
+        "slice_filter_params": slice_filter_params,
+        "slice_axis": "axial",
+        "multichannel": False
+    }
+    # Get Training dataset
+    bids_df = BidsDataframe(loader_params, __tmp_dir__, derivatives=True)
+    ds = imed_loader.load_dataset(bids_df, **loader_params)
+
+    print('\tNumber of loaded patches: {}'.format(len(ds)))
```

Comment:
I recommend use loguru.logger as most of the other underlying modules are already using so the output will be more consistent and auto time stamped. 

Also, I recommend to use [`f`string format ](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python)for easier readability. 

![image](https://user-images.githubusercontent.com/6720068/142910478-4bc59c22-2c8e-4395-8c98-d0b26404050b.png)

Reply:
Again, I'm building on previous code here. So I duplicated the previous test. I can make the changes for both tests, or we could open a separate PR to clean up "print" in all our tests at once.

Reply:
Good idea. I thought that was done but it seems we definitely missed some. I am going to write up a ticket to review this across the code base.

Reply:
NPLY-80/NPLY-84